pose-detection/src/blazepose/detector.ts

```diff
@@ -329,7 +322,7 @@ export class BlazeposeDetector extends BasePoseDetector {
       tf.dispose(landmarkResult);
       tf.dispose([imageTensor, imageValueShifted]);
 
-      return null;
+      return {actualLandmarks: [], auxiliaryLandmarks: [], poseScore};
```

Comment:
Looks like propagating `null` instead of `[]` is better in general case cause empty array is can be a valid output in some other cases.